scripts/perftune.py

```diff
@@ -643,6 +643,35 @@ def __intel_irq_to_queue_idx(self, irq):
         else:
             return sys.maxsize
 
+    def __mlx_irq_to_queue_idx(self, irq):
+        """
+        Return the HW queue index for a given IRQ for Intel NICs in order to sort the IRQs' list by this index.
```

Comment:
The comment refers Intel which the function name implies Mellanox.

Reply:
It looks half baked anyway - I should probably wait with a review till it's ready... ;)